ice40/cells/plb/plb.pb_type.xml

```diff
@@ -164,14 +164,26 @@
  </pb_type>
 
  <pb_type name="BLK_IG-LUTFF" num_pb="8">
-  <xi:include href="../lutff/lutff.pb_type.xml" xpointer="xpointer(pb_type/child::node())"/>
+   <xi:include href="../lutff/lutff.pb_type.xml" xpointer="xpointer(pb_type/child::node())"/>
+   <metadata>
+     <meta name="hlc_cell">lutff</meta>
```

Comment:
Did you intend to leave the hlc_cell annotation?

Reply:
I will remove in follow up PR.